src/test/java/weblogic/logging/exporter/ResultTest.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (c) 2019, Oracle and/or its affiliates. All rights reserved.
+ *
+ * Licensed under the Universal Permissive License v 1.0 as shown at http://oss.oracle.com/licenses/upl.
+ */
+
+package weblogic.logging.exporter;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+@DisplayName("Test the Result object")
+public class ResultTest {
+
+  private static final String EXPECTED_STRING =
+      "Result{response='good', status=1, successful=true}";
+
+  @DisplayName("Check constructor and getters")
+  @Test
+  public void testConstructorAndGetters() {
+    Result result = new Result("good", 1, true);
+
+    assertEquals("good", result.getResponse());
+    assertEquals(1, result.getStatus());
+    assertEquals(true, result.isSuccessful());
+  }
+
+  @DisplayName("Check toString() works")
```

Comment:
I usually don't bother unit testing the toString() functionality, as it isn't particularly relevant to the functionality of the code, and is liable to change any time somebody modifies the class. That makes tests on it inherently brittle.